persistence/src/commonMain/kotlin/com/wire/kalium/persistence/dao/message/MessageEntity.kt

```diff
@@ -251,7 +252,9 @@ data class MessagePreviewEntity(
     val content: MessagePreviewEntityContent,
     val date: String,
     val visibility: MessageEntity.Visibility,
-    val isSelfMessage: Boolean
+    val isSelfMessage: Boolean,
+    val senderUserId: QualifiedIDEntity,
+    val selfUserId: UserIDEntity?,
```

Comment:
I'm not sure about the benefits of including self user ID in every message when it will be always the same for all messages.
The check is self removed was removed/added is needed only for the UI to display the correct copy; therefore it can be left to the UI to do this comparison

Reply:
Removed 😄